source/tutorial/backend.txt

```diff
@@ -53,22 +53,28 @@ You'll need to have the following before you can begin this tutorial:
   for MongoDB Atlas and create one for free
   <https://account.mongodb.com/account/register?tck=docs_realm>`__.
 
+Also note that this tutorial doesn't use the Deployment Draft workflow. Unless you have disabled 
+Deployment Drafts, you may need to save your changes to a draft and then deploy them manually.
+
 Create a New {+app+}
 ----------------
 
-First, you'll need to create an application in {+service-short+}. To create the app:
+First, you'll need to create an application in {+service-short+}. If you see a prompt about 
+Aplication Guides, select :guilabel:`Close Guides`.
+
+To create the app:
 
 1. Open your Atlas project at `cloud.mongodb.com <https://cloud.mongodb.com?tck=docs_realm>`__.
 
-#. Click :guilabel:`{+service-short+}` in the top navigation.
+#. In the top navigation, click :guilabel:`{+service-short+}`.
 
 #. Click :guilabel:`Create a New App`.
 
 #. Name the app ``github-tracker-tutorial``.
 
 #. Select your cluster to link it to the new app.
 
-#. Click :guilabel:`Create`.
+#. Click :guilabel:`Create App Service`.
```

Comment:
it says 'Create App Service'? that's odd b/c usually we say "App Services App" or "App".

Reply:
Yep! It does. lol
![Screen Shot 2022-11-02 at 11 36 36 AM](https://user-images.githubusercontent.com/115574589/199596206-5b577946-6796-4808-b20e-93cfd085bd0b.png)